src/utils/grammars/michelsonParsingUtil.ts

```diff
@@ -0,0 +1,17 @@
+const nearley = require("nearley");
```

Comment:
This file should probably be in /src/chain/tezos

Reply:
Got it.